test/test_save.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+import py.path
+import rasterio
+
+from satsense import FeatureVector, Image
+from satsense.extract import extract_feature
+from satsense.features import HistogramOfGradients
```

Comment:
You may want to use a dummy feature here to keep a loose coupling between the components of the package and to prevent confusingly high coverage of the HoG feature.